pkg/manager/impl/execution_manager_test.go

```diff
@@ -2932,7 +2932,46 @@ func TestGetExecutionConfig(t *testing.T) {
 	execConfig, err := executionManager.getExecutionConfig(context.TODO(), &admin.ExecutionCreateRequest{
 		Project: workflowIdentifier.Project,
 		Domain:  workflowIdentifier.Domain,
+		Spec:    &admin.ExecutionSpec{},
```

Comment:
we always validate that incoming ExecutionCreateRequests have non-nil specs, so this change/assumption should be fine